README.md

```diff
@@ -32,8 +32,12 @@ to setup database from docker.
 3. To connect to the local mysql database instance with `mysql -u root -p mpv -h 127.0.0.1 -P 3306`
 4. Password for development is `password`
 
-Create your `config.py` file to look like:
+### Configuration:
```

Comment:
I think the addition of the config section to the README definitely makes sense. With what we have above on #3 under `Installing`, I think it might make sense to consolidate the two together. I'll take care of it. :-)